core/server/common/src/main/java/alluxio/cli/validation/HdfsValidationTask.java

```diff
@@ -54,20 +55,25 @@ public boolean validate(Map<String, String> optionsMap) {
       return true;
     }
     if (!validateHdfsSettingParity(optionsMap)) {
-      System.err.format("Hdfs setting do not match.");
+      System.err.format("Hdfs setting do not match.%n");
       return false;
     }
     return true;
   }
 
   /**
-   * Checks whether this validation task should be skipped.
-   *
-   * @return true if underFS is NOT HDFS, false otherwise
+   * {@inheritDoc}
    */
-  protected boolean shouldSkip() {
-    String underFSURI = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS).toLowerCase();
-    return !underFSURI.startsWith("hdfs://");
+  @Override
+  public boolean shouldSkip() {
+    String scheme =
+        new AlluxioURI(Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS)).getScheme();
+    if (scheme == null || !scheme.startsWith("hdfs")) {
+      System.err.format("Root underFS is %s, not HDFS. Skipping validation for HDFS properties.%n",
```

Comment:
why print to `System.err`?

Reply:
In other parts of validation script, `System.err` is used. I'm using `System.err` to conform to the convention in this module.

Reply:
It looks like the convention is to use `System.out` for normal control flow, and `System.err` when checks fail

https://github.com/Alluxio/alluxio/blob/master/core/server/common/src/main/java/alluxio/cli/ValidateEnv.java#L229
https://github.com/Alluxio/alluxio/blob/master/core/server/common/src/main/java/alluxio/cli/ValidateEnv.java#L231
https://github.com/Alluxio/alluxio/blob/master/core/server/common/src/main/java/alluxio/cli/ValidateEnv.java#L244

Skipping an unnecessary check doesn't mean anything is wrong, so I think the convention is to use `System.out`. @bf8086 is this accurate?

Reply:
I think @aaudiber  is correct. I'll make the change. Thanks for the catch.

Reply:
That's right `System.err` is only for error messages.